### DISCUSSION TITLE
武昌首义学院（嘉鱼校区）

### DISCUSSION THREAD

**Discussion by Yilimmilk** [CollegesChat/university-information#97] [University]
# 武昌首义学院嘉鱼校区

 ## Q: 宿舍是上床下桌吗？

 - A1: 是

 ## Q: 教室和宿舍有没有空调？

 - A1: 有

 ## Q: 有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远？

 - A1: 有独立卫浴，有热水，刷校园卡就可以用

 ## Q: 有早自习、晚自习吗？

 - A1: 目前大一，有早自习，无晚自习

 ## Q: 有晨跑吗？

 - A1: 取决于辅导员好不好，咱们辅导员不错，每次都没让我们去，其它班大部分都要出早操

 ## Q: 每学期跑步打卡的要求是多少公里，可以骑车吗？

 - A1: 每周三次三公里，具体取决于老师要求，有的班少一些有的班多一些   不能骑车

 ## Q: 寒暑假放多久，每年小学期有多长？

 - A1: 寒暑假两个月左右，目前无小学期

 ## Q: 学校允许点外卖吗，取外卖的地方离宿舍楼多远？

 - A1: 可以点外卖，只能送到东门，离宿舍楼300米左右

 ## Q: 学校交通便利吗，有地铁吗，在市区吗，不在的话进城要多久？

 - A1: 嘉鱼校区位于咸宁市嘉鱼县，十分不方便，没地铁，回武汉只能坐大巴，时间为一小时四十分钟+，无高铁动车火车，距离武汉主校区过于远，坐校车需要一小时二十分钟，招生简章上写的四十分钟是假的

 ## Q: 宿舍楼有洗衣机吗？

 - A1: 有洗衣机，滚筒式，标准洗4元一次

 ## Q: 校园网怎么样？

 - A1: 由于嘉鱼校区属于新建校区，电信垄断，目前校园网处于测试阶段，带宽目测为30M左右，而且信号极其不好，稳定性极差，使用电信飞young客户端。宿舍宽带可选50M或100M，也是必须使用飞young客户端，稳定性很差，王者经常460。但可以在淘宝上购买破解路由器，不要信任何所谓“学长学姐”的话，买他们的产品，会坑你。

 ## Q: 每天断电断网吗，几点开始断？

 - A1:无，限电800W，但空调插座不限电，可以牵一个插座出来

 ## Q: 食堂价格贵吗，会吃出异物吗？

 - A1: 贵，大部分店家不好吃，目前感觉常青麦香园，煲仔饭，鸡肉饭比较好吃，店家经常改名字，所以这一条不准确，自己踩坑吧哈哈哈哈哈

 ## Q: 洗澡热水供应时间？

 - A1: 早中晚都有固定供应，具体取决于学校

 ## Q: 校园内可以骑电瓶车吗，电池在哪能充电？

 - A1: 无车，不允许买电瓶车，无充电处

 ## Q: 宿舍限电情况？

 - A1: 800W

 ## Q: 通宵自习有去处吗？

 - A1: 无

 ## Q: 大一能带电脑吗？

 - A1: 可以

 ## Q: 学校里面用什么卡，饭堂怎样消费？

 - A1: 校园卡可微信支付宝充值使用，食堂支持校园卡和手机支付

 ## Q: 学校会给学生发银行卡吗？

 - A1: 会，中国工商银行的，卡挺好看的

 ## Q: 学校的超市怎么样？

 - A1: 质量较差，东西较贵，建议淘宝

 ## Q: 学校的收发快递政策怎么样？

 - A1: 所有快递都在实验楼有快递点

 ## Q: 学校里面的共享单车数目与种类如何？

 - A1: 无共享单车

 ## Q: 现阶段学校的门禁情况如何？

 - A1: 出去需要写请假条，毕竟学校在建，可以从工地旁边的门，窗溜出去，挺简单。进来只用看请假条，其他无限制

 ## Q: 宿舍晚上查寝吗，封寝吗，晚归能回去吗？

 - A1: 有时查寝，每天晚上十一点半关门，晚归开不开门取决于你和宿管阿姨的关系

 ## 自由补充部分

 嘉鱼校区环境十分不好，绿化不足，校园设施不完整，无娱乐，校园整体建筑风格较为一致，第一眼觉得还可以，后续有点审美疲劳，感觉挺丑的，真的不大气，但宿舍条件相比武汉主校区好很多。目前2021年六月，学校可以去的设施就只有足球场，篮球场，宿舍，教学楼，实验楼，行政楼，图书馆还没建好，只有临时图书馆（就是从实验楼中空了几个房间出来），毕竟目前还在建，可以称为工地大学，但嘉鱼县这个地方挺适合养老的，附近的二乔公园环境确实好，相信这个校区以后发展还可以吧。

**Comment by kunpeng9** [comment]
而且信号极其不好，稳定性极差，使用电信飞young客户端，
15年高中毕业后上的这个学院，被学院宽带坑死，时不时掉线，非要用客户端出乱七八糟的各种驱动问题，老是有错误代码，还有什么问题，
经常没信号或者突然掉速，或者突然断网。
重装好多次这个客户端，然后又要重启，累死了。
导致我少尝试很多科技分支。
学校收费1万到2万之间，这么贵的学费，服务如此差劲，基础设施连个宽带都搞不好，
我上学的时候机械专业的，机床居然还有五六十年代的，好久没换了，我毕业之后一两年之后才换的
